content/sensu-go/6.5/observability-pipeline/observe-transform/mutators.md

```diff
@@ -119,6 +200,29 @@ As a result, executable scripts (like plugins) located in `/etc/sensu/plugins` w
 This allows `command` attributes to use “relative paths” for Sensu plugin commands (for example, `"command": "check-http.go -u https://sensuapp.org"`).
 {{% /notice %}}
 
+## JavaScript mutators
+
+Mutators that use JavaScript are an efficient alternative to pipe mutators, which fork a process on each invocation.
+JavaScript mutators are evaluated by the [Otto JavaScript VM][19] as JavaScript programs, which enables greater mutator throughput at scale.
+
+JavaScript mutators do not require you to return any value &mdash; you can either also mutate the events that are passed to the mutator.
```

Comment:
This sentence is a bit off

Reply:
(line 208)

Reply:
Yeah, that's messy. Thank you for catching it.